platform/darwin/src/NSValue+MGLStyleEnumAttributeAdditions.h

```diff
@@ -0,0 +1,150 @@
+// This file is generated.
+// Edit platform/darwin/scripts/generate-style-code.js, then run `make style-code-darwin`.
+
+#import <Foundation/Foundation.h>
+
+#import "MGLFillStyleLayer.h"
+#import "MGLLineStyleLayer.h"
+#import "MGLSymbolStyleLayer.h"
+#import "MGLCircleStyleLayer.h"
+#import "MGLRasterStyleLayer.h"
+#import "MGLBackgroundStyleLayer.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NSValue (MGLStyleEnumAttributeAdditions)
+
+#pragma mark Runtime Styling enumerations
```

Comment:
A documentation section name (that is,  a`mark` in a public header) should describe a task, in title case for consistency:

```objc
#pragma mark Working with Style Layer Attribute Values
```